ios/FluentUI.Demo/FluentUI.Demo/Demos/IndeterminateProgressBarDemoController.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class IndeterminateProgressBarDemoController: DemoController {
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		//Indeterminate Progress Bar
+		let progressBar = IndeterminateProgressBarView()
+		container.addArrangedSubview(progressBar)
+		
+		//Set the constraints
+		progressBar.translatesAutoresizingMaskIntoConstraints = false
+		NSLayoutConstraint.activate([
+			progressBar.topAnchor.constraint(equalTo: self.view.topAnchor, constant: 20),
+			progressBar.leadingAnchor.constraint(equalTo: self.view.leadingAnchor, constant:0),
+			progressBar.trailingAnchor.constraint(equalTo: self.view.trailingAnchor, constant:0),
+			progressBar.heightAnchor.constraint(equalToConstant: 4.0)
```

Comment:
i don't think it is client's responsibility to setup the height of the control. isn't the design intend to have the progressbar always be 4 ? similar to Separator control in fluent, the intrinsiccontentsize and sizethatfits should determine what the control size is.